pkg/gotohelm/testdata/src/example/sprig/sprig.go

```diff
@@ -15,9 +15,26 @@ func Sprig() map[string]any {
 	return map[string]any{
 		"empty":   empty(),
 		"default": default_(),
+		"unset": unset(),
 	}
 }
 
+func unset() []map[string]int {
+	m1 := map[string]int{"0": 0, "1": 1, "2": 2}
+	m2 := map[string]int{"0": 0, "1": 1, "2": 2}
+	m3 := map[string]int{"0": 0, "1": 1, "2": 2}
+	m4 := map[string]int{"0": 0, "1": 1, "2": 2}
+
+	delete(m2, "0")
```

Comment:
I'm not sure how `delete` transpilation maps to `unset` helm template.

Reply:
Golang's `delete` and sprig's `unset` are equivalent functions, no? They both remove a key from a dictionary/map.

Reply:
THX for explaining.